handbook/engineering/web/frontend-platform/index.md

```diff
@@ -15,6 +15,35 @@ The Frontend Platform team (part of the Web org) defines and maintains the stand
     - [Felipe Janer](../../../../company/team/index.md#felipe-janer-he-him)
     - [Valery Bugakov](../../../../company/team/index.md#valery-bugakov-he-him)
 
+## Mission
+
+To empower all users and Sourcegraph frontend developers to achieve maximum efficiency and effectiveness, by enabling and building a first-class web experience.
+
+## Vision
+
+Using and developing Sourcegraph is effortless.
+- We have a frictionless developer experience that empowers our frontend engineers to achieve the maximum value from the time spent on development.
+- We have an intuitive, performant, and consistent user experience that makes Sourcegraph an indispensable part of every developer workflow.
+
+## Responsibilities
+
+1. Frontend platform:
+   - Creating and maintaining Wildcard Component Library.
```

Comment:
```suggestion
   - Creating and maintaining the Wildcard Component Library.
```

Is it possible to link to some resource about Wildcard COmponent Library? Even if it's just a Figma link for now.

Reply:
@AlicjaSuska Do we have anything we can link to?

Reply:
I'm going to go ahead and land this PR as-is, but will add a link as soon as we have something appropriate to link to :-)